platform-api-client-gwt/che-core-client-gwt-git/src/main/java/org/eclipse/che/api/git/gwt/client/GitServiceClient.java

```diff
@@ -431,6 +431,16 @@ void diff(ProjectConfigDto project,
               AsyncRequestCallback<String> callback);
 
     /**
+     * Get the file content from specified revision or branch.
+     *
+     * @param project project (root of GIT repository)
```

Comment:
Reformat it in accordance to https://github.com/codenvy/codenvy-dev-resources/wiki/Formatting#javadoc